src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -354,16 +356,22 @@ private static bool CompileProject(ProjectContext context, CompilerCommandApp ar
             }
             else if (!string.IsNullOrEmpty(context.ProjectFile.TestRunner))
```

Comment:
Wow this is super hacky. Why does compile know about test runners.... We need to decouple that.

/cc @piotrpMSFT